src/DssSpell.t.base.sol

```diff
@@ -425,7 +425,7 @@ contract DssSpellTestBase is DSTest, DSMath {
             line:         0,
             dust:         15 * THOUSAND,
             pct:          400,
-            mat:          1120000,
```

Comment:
We can keep it as it's the target `mat` (we can cleanup during the housekeeping eventually)